docs/UserGuide.md

```diff
@@ -1,125 +1,151 @@
+shrink = false # whether allow to shrink cache ways in best effort pool
 max_allowed_shared = 10 # max allowed workload in shared pool, default is 10
 guarantee = 10
 besteffort = 7
 shared = 2
 ```
 
-- OSGroup: cache ways reserved for operate system usage.
-- InfraGroup: infrastructure tasks group, user can specify task binary name
-              the cache ways will be shared with other worklaod
+Comments of the directives in the above conf:
 
-Besides, define cache pool, cache allocation will happened in the pools.
-
-- shrink: whether shrink cache ways which already allocated to workload in
-          besteffort pool.
-- max_allowed_shared: max allowed workloads in shared cache pool
-- guarantee: allocate cache for workload max_cache == min_cache > 0
-- besteffort: allocate cache for workload max_cache > min_cache > 0
-- shared: allocate cache for workload max_cache == min_cache = 0
+* OSGroup: cache ways reserved for operating system usage.
+* InfraGroup: infrastructure tasks group, user can specify task binary name
+              the cache ways will be shared with other workloads
+* CachePool: cache allocation will happened in the pools.
+    - shrink: whether shrink cache ways which already allocated to workload in
+            "besteffort" pool.
+    - max_allowed_shared: max allowed workloads in shared cache pool
+    - guarantee: allocate cache for workload max_cache == min_cache > 0
+    - besteffort: allocate cache for workload max_cache > min_cache > 0
+    - shared: allocate cache for workload max_cache == min_cache = 0
 
 On a host which support max 20 cache ways, for this configuration file,
 we will have follow cache bit mask layout:
-
 ```
-
 OSGroup:    0000 0000 0000 0000 0001
 InfraGroup: 1111 1111 1111 1111 1110
 ```
+
 Available CBM in Cache Pools initially:
 ```
 guarantee:  0000 0000 0111 1111 1110
 besteffort: 0011 1111 1000 0000 0000
 shared:     1100 0000 0000 0000 0000
 ```
 
-## Create a workload
+### Prepare the credentials
 
-A workload could be a running task(s) or some cpus which want to allocate
-cache for them.
+For security considerations, RMD daemon runs the RESTful API service as a
+dedicated Unix user 'rmd' and performs privileged operations (PAM
+authentication or resctrl) as Unix 'root' user. RMD users need to prepare
+several things by themselves.
+
+1. Create user and group 'rmd' in the target Linux system
 
-The task(s)/cpus should be valided, RMD will fail your request they are
-invalid.
+    ```shell
+    $ sudo <TODO>
+    ```
+2. Ensure the following packages are installed on your target system for PAM
 
-Besides you need to specify what policy of the workload will be usage.
+    Debian or Ubuntu:
+    ```shell
+    $ sudo apt-get install openssl libpam0g-dev db-util
+    ```
+    Redhat(s)
+    ```shell
+    $ sudo dnf install openssl pam-devel db4-utils
+    ```
+3. Besides host Unix credentials, RMD can also use dedicated credentials setup
```

Comment:
Better to mention this is optional, only use pam as auth should require this step